web_src/js/features/common-global.js

```diff
@@ -118,7 +119,7 @@ export function initGlobalCommon() {
   $('.tabable.menu .item').tab();
 
   $('.toggle.button').on('click', function () {
-    $($(this).data('target')).slideToggle(100);
+    toggleElem($($(this).data('target')));
```

Comment:
the original code has a `100`ms but new code hasn't.

Reply:
In history, some people like the animation, while some others do not like.

But the animation (including layout & css) are quite hacky and buggy there. It's good to remove the animation this time.

In the future, people who like the animation could make a clean solution for that.